src/main/java/com/telefonica/iot/cygnus/backends/orion/OrionBackendImpl.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-cygnus (FI-WARE project).
+ *
+ * fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-cygnus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-cygnus. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+ */
+
+package com.telefonica.iot.cygnus.backends.orion;
+
+import com.telefonica.iot.cygnus.backends.http.HttpBackend;
+import com.telefonica.iot.cygnus.backends.http.JsonResponse;
+import com.telefonica.iot.cygnus.log.CygnusLogger;
+import java.util.ArrayList;
+import org.apache.http.Header;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.message.BasicHeader;
+
+/**
+ *
+ * @author frb
+ */
+public class OrionBackendImpl extends HttpBackend implements OrionBackend {
+    
+    private static final CygnusLogger LOGGER = new CygnusLogger(HttpBackend.class);
+    
+    /**
+     * Constructor.
+     * @param orionHost
+     * @param orionPort
+     * @param ssl
+     */
+    public OrionBackendImpl(String orionHost, String orionPort, boolean ssl) {
+        super(new String[]{orionHost}, orionPort, ssl, false, null, null, null, null);
+    } // StatsBackendImpl
+
+    @Override
+    public JsonResponse subscribeContext(String subscription, boolean xAuthToken, 
+            String token) throws Exception {
+        
+        // create the relative URL
+        String relativeURL = "/v1/subscribeContext";
```

Comment:
That relativeURL belongs to Orion, for that reason must be /v1/subscribeContext :)